### DISCUSSION TITLE
Tiddler with Canonical URI: The Text Filed overflown

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6352] [Q&A] [answered]
Look at the below image: a tiddler with _canonical_uri: 
The blue box which indicates the text field has overflown from both side! Is this intentional or a css issue?

![img_746_chrome](https://user-images.githubusercontent.com/830394/146599424-d3717f53-bf23-4413-bafd-172f93286d96.png)

**Comment by Jermolene** [comment, marked as answer]
It is intentional – the border bleeds horizontally outside the highlight box so that the text within lines up with the surrounding text.
